src/com/spikes2212/utils/InvertedConsumer.java

```diff
@@ -0,0 +1,39 @@
+package com.spikes2212.utils;
+
+import java.util.function.Consumer;
+

```

Comment:
do you think this should be in generic.utils or only in utils?

Reply:
in think it should be in subsytem's utils

Reply:
so move it there than :)